index.js

```diff
@@ -40,11 +40,7 @@ function readDirRecurse (rootDir: string, opts: Object): Promise {
       .on('data', (item) => {
         let excluded = false
         if (opts && opts.excludeDirectories) {
-          for (const dir of opts.excludeDirectories) {
-            if (item.path.indexOf(dir) !== -1) {
-              excluded = true
-            }
-          }
+          excluded = opts.excludeDirectories.filter((dir) => item.path.indexOf(dir) !== -1).length
```

Comment:
Let's not mix types--if you declare it as a boolean above, make this a boolean compare:
`.length > 0` or `.length !== 0`

Reply:
Updated :+1: